compatibility-test/setup_bindings_inner.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Generate bindings for all languages inside container
+LIBRARY="/workspace/target/debug/libcompatibility_test.so"    
+uniffi-bindgen generate $LIBRARY --library --language python --out-dir tmp/python
+uniffi-bindgen generate $LIBRARY --library --language kotlin --out-dir tmp/kotlin
+uniffi-bindgen generate $LIBRARY --library --language swift --out-dir tmp/swift
+uniffi-bindgen-cs $LIBRARY --library --out-dir tmp/cs/UniffiCS/generated
+uniffi-bindgen-go $LIBRARY --library --out-dir tmp/go/generated
+uniffi-bindgen-cpp $LIBRARY --library --out-dir tmp/cpp
```

Comment:
Nitpick: is there a reason why C# and Go don't follow the same directory structure here?

Reply:
I just copied the directory structure from generator projects. For C# I think the nesting is optional. For Go its mandatory, because tests import a module based on that directory.

Reply:
Ah, since it's mandatory for Go, guess doesn't make much sense to try to make it uniform :+1: